product/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from core      import TimeStampModel
+
+# Create your models here.
+class Main_Category(models.Model):
+    name=models.CharField(max_length=50)
+    
+    class Meta:
+     db_table='main_categories'
+     
+    def __str__(self):
+            return self.main_categories_name
+        
+class Sub_category(models.Model):
+    sub_category_name = models.CharField(max_length=50)
+    Main_Category     = models.ForeignKey("Main_category",on_delete=models.CASCADE)
+    
+    class Meta:
+     db_table='sub_categories'
+     
+class Product(TimeStampModel):
+    sub_category      = models.ForeignKey('Sub_categories', on_delete=models.CASCADE)
+    products_name     = models.CharField(max_length=50)
+    products_contents = models.TextField()
+    
+    class Meta:
+     db_table='products'
+
+class Product_option(TimeStampModel):
+    products_id      = models.ForeignKey('Product', on_delete=models.CASCADE)
+    sizes_mL         = models.CharField(max_length=20)
+    products_img_url = models.URLField()
+    price            = models.DecimalField(max_length=20)
+    include_pump     = models.BooleanField()
+    size_contents    = models.TextField()
```

Comment:
- 바뀐 field 이름 확인해 주세요.
- Decimal Field의 필수 옵션 값들이 있습니다. 확인해 보고 추가해 주세요.
```suggestion
class ProductOption(TimeStampModel):
    product         = models.ForeignKey('Product', on_delete=models.CASCADE)
    size_mL         = models.CharField(max_length=20)
    image_url       = models.URLField()
    price           = models.DecimalField()
    is_include_pump = models.BooleanField()
    content         = models.TextField()
```